<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SCD - v0.1</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/resources/scd.svg</normaloff>:/icons/resources/scd.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSystem">
    <property name="title">
     <string>System</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menuOperation">
    <property name="title">
     <string>Operation</string>
    </property>
    <addaction name="actionMode1"/>
    <addaction name="actionMode2"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionSCD"/>
   </widget>
   <addaction name="menuSystem"/>
   <addaction name="menuOperation"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/start.svg</normaloff>:/icons/resources/start.svg</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>This will start the system</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/stop.svg</normaloff>:/icons/resources/stop.svg</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>This will stop the system</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionMode1">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/mode1.svg</normaloff>:/icons/resources/mode1.svg</iconset>
   </property>
   <property name="text">
    <string>Mode 1</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionMode2">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/mode2.svg</normaloff>:/icons/resources/mode2.svg</iconset>
   </property>
   <property name="text">
    <string>Mode 2</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionSCD">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/scd.svg</normaloff>:/icons/resources/scd.svg</iconset>
   </property>
   <property name="text">
    <string>SCD</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
